miner/worker.go

```diff
@@ -322,10 +322,11 @@ func (w *worker) close() {
 }
 
 func (w *worker) createTxCmp() func(tx1 *types.Transaction, tx2 *types.Transaction) int {
-	currencyComparator := currency.NewComparator()
+	// TODO specify header & state
+	currencyManager := currency.NewManager(nil, nil)
```

Comment:
Which header and state should be used here? Id imagine the parent of `w.current` is the best candidate. If at all possible we should not merge without fixing this.